docs/index.rst

```diff
@@ -11,7 +11,6 @@ Welcome to Foreman Ansible Modules' documentation!
    :caption: User documentation
 
    README
-   Changelog <CHANGELOG>
```

Comment:
The simple version would be to move this to below `cvmanager` or possibly past `releasing`, but without adding the extra toctree.